fetch.bs

```diff
@@ -4575,6 +4579,37 @@ steps. They return a <a for=/>response</a>.
    <var>fetchParams</var>.
   </ol>
 
+ <li>
+  <p>If <var>response</var>'s <a for=response>status</a> is 421, then:
+
+  <ol>
+   <li>
+    <p>If the ongoing fetch is <a for=fetch>terminated</a>, then:
+
+    <ol>
+     <li><p>Let <var>aborted</var> be the termination's aborted flag.
+
+     <li><p>If <var>aborted</var> is set, then return an <a>aborted network error</a>.
+
+     <li><p>Return a <a>network error</a>.
+    </ol>
+
+   <li>
+    <p>If <var>actualResponse</var>'s <a for=response>status</a> is 421,
+     <var>request</var>'s <a for=request>body</a> is non-null, and
+     <var>request</var>'s <a for=request>body</a>'s <a for=body>source</a> is
+      null, then return a <a>network error</a>.
+
+   <li>
+     <p>Set <var>bypassConnectionPool</var> to true.
+
+   <li>
+     <p>Set <var>response</var> to the result of running
+      <a>HTTP-network-or-cache fetch</a> given <var>fetchParams</var> and
+      <var>bypassConnectionPool</var>.
```

Comment:
This also needs to pass isAuthenticationFetch. Not sure what value it should be.

Reply:
Done. Just reused. BTW, no one call HTTP-network-or-cache fetch with isAuthenticationFetch parameter...

Reply:
https://fetch.spec.whatwg.org/#ref-for-concept-http-network-or-cache-fetch%E2%91%A0 is the caller.

Reply:
I pushed some formatting nits to this PR. I'd appreciate your review. I also still don't understand something about the tests and left a comment there. Apart from that I think this all turned out great.